### PR TITLE
#313

### PR DESCRIPTION
**Pull request by igaidai4uk**

### CODE REVIEW

**Comment by doom369** [comment]
Also please put comment in commit message. Not only ticket number.

File: server/core/src/main/java/cc/blynk/server/core/processors/EventorProcessor.java [hunk 1]
```diff
@@ -79,6 +80,8 @@ public void process(Session session, DashBoard dash, byte pin, PinType type, Str
                 }
             }
         }
+
+        globalStats.mark(EVENTOR);
```
Comment (igaidai4uk) [inline server/core/src/main/java/cc/blynk/server/core/processors/EventorProcessor.java]: Both execute methods? If yes, then why execute for pin action is static?

Comment (doom369) [inline server/core/src/main/java/cc/blynk/server/core/processors/EventorProcessor.java]: We need to understand how many events Eventor triggers. I think you can find right place for that,

**Comment by doom369** [comment]
Please squash your commit. 1 Feature - 1 commit.

**Pull request closed without merging**